src/test/java/org/folio/search/sample/SampleInstances.java

```diff
@@ -9,7 +11,7 @@
   private static final Instance SEMANTIC_WEB = readSampleInstance("semantic_web_primer.json");
 
   public static Instance getSemanticWeb() {
-    return SEMANTIC_WEB;
+    return OBJECT_MAPPER.convertValue(SEMANTIC_WEB, Instance.class);
```

Comment:
Isn't `SEMANTIC_WEB` already `Instance`?

Reply:
Yes, it is. This convert needed to make a copy of the instance, so that the original instance is not changed from the outer code.